lib/zssAuth.js

```diff
@@ -8,37 +8,41 @@
   Copyright Contributors to the Zowe Project.
 */
 
+const DEFAULT_EXPIRATION_MS = 3600000 //hour;
+
 function ZssAuthenticator(pluginDef, pluginConf, serverConf) {
   this.authPluginID = pluginDef.identifier;
+  this.sessionExpirationMS = DEFAULT_EXPIRATION_MS; //ahead of time assumption of unconfigurable zss session length
 }
 
 ZssAuthenticator.prototype = {
 
-  getStatus(sessionState) {
-    return {  
-      authenticated: !!sessionState.authenticated, 
-      username: sessionState.zssUsername
-    };
+  getStatus(request, sessionState) {
```

Comment:
I think this would suffice: (NOT TESTED)
 ```
 getStatus(request, sessionState) {
    return this._authenticateOrRefresh(request, sessionState, true).catch(e => {
        //dont un-auth or delete cookie... perhaps this was a network error.
        //Let session expire naturally if no success
        resolve({ authenticated: false }) 
      });
  },
```